modules/doc/config.yml

```diff
@@ -40,29 +40,29 @@ Extensions:
         repo: https://github.com/idaholab/moose
         google-cse: 000781241768298771085:pedtnlxdkx8
         menu:
+            Training:
+                Comming Soon: /empty.md
             Getting Started:
                 Installing MOOSE: getting_started/installation/index.md
                 Creating an Application: getting_started/new_users.md
                 Examples and Tutorials: getting_started/examples_and_tutorials/index.md
                 NCRC Applications: help/inl/applications.md
-                Supplemental Resources: getting_started/supplemental_resources/index.md
                 INL HPC Services: help/inl/index.md
+                Supplemental Resources: getting_started/supplemental_resources/index.md
+                Frequently Asked Questions: help/faq/index.md
+                Troubleshooting: help/troubleshooting.md
+                Help Forum: https://github.com/idaholab/moose/discussions
+                Contact Us: help/contact_us.md
             Documentation:
                 Application Usage: application_usage/index.md
                 Application Development: application_development/index.md
                 Framework Development: framework_development/index.md
                 Modules: modules/index.md
-                NCRC Applications: help/inl/applications.md
```

Comment:
this should stay?

Reply:
I suppose we can leave it. Its duplicated under Getting Started, so I figure meh.

Reply:
added it back.